ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -844,76 +892,207 @@ func (pg *DEXOnboarding) HandleUserInteractions() {
 				pg.currentStep = onboardingChooseServer
 			}
 		case onboardingChooseServer, onBoardingStepAddServer:
-			var serverURL string
-			var serverCert []byte
+			if pg.isLoading {
+				break
+			}
```

Comment:
I think check this early and return if some background process is taking place, instead of checking in the switch cases. The first case doesn't check but that should also never run if some background process is taking place.